### PR TITLE
[Feature] : Work component

### PR DESCRIPTION
**Pull request by SahilVermaFED**
## Description

-   [Ticket Link](#) (if any)
-   This PR includes :
    1. Custom CSS styles, addressing font import issues and introducing color names, border radius, width, font size, background color, line height, and padding .    
    2. New assets (logo and images), removes the default React logo .
    3. GSAP for scroll handling, leading to the creation of 'scrollHandlerUtils.jsx'

## UI screenshot or Video

https://github.com/primathonacademy/april-batch-live-project/assets/131689244/869ad761-f170-46bf-8f55-1f57804b7942

### General:

-   [x] **PR Size :** Confirming PR size < 20 files or pre-approved if > 20 files.
-   [x] **Self Review :** Confirming I have self-reviewed my PR
-   [x] **Peers Review :** - [ ] @Dev 1
-   [x] Confirming I have added all sensitive information-related should be inside .env files

### Client:

-   [x] **Console :** Confirming no console errors in the UI for this update.

-   [x] **Import order :** Confirming correct import order.

```
# External Import
import React, { useState, useEffect } from 'react';
import { useParams } from 'react-router-dom';
import { useDispatch, useSelector, shallowEqual } from 'react-redux';

# Internal Import
import { } from '../../Api/ApiRoutes';
import { } from '../../constants/AppConstant';
```

-   [ ] **Re-renders :** Confirming minimal re-renders for new React components.

-   [ ] **Network calls :** Confirming efficient network/API calls, minimized as needed.

-   [ ] **Util function comments :** Confirming comprehensive comments for utility functions.

-   [x] **Image optimization :** Confirming I have checked that all images are optimized. If not, please use https://tinyjpg.com/ or https://vecta.io/nano.

-   [ ] **Dead code :** Confirming I have removed all comments code which is of no use OR written why I have kept the commented code.

### Confirming different viewport testing of the entire flow:

-   [x] Tested on 280px viewport
-   [x] Tested on 320px viewport
-   [x] Tested on 375px viewport
-   [x] Tested on 425px viewport
-   [x] Tested on 768px viewport
-   [x] Tested on 1024px viewport
-   [x] Tested on 1440px viewport
-   [x] Tested on 2560px viewport

### End-to-end testing:

-   [x] **Happy flow :** Successfully tested end-to-end happy flow, all working as expected.
-   [x] **Error flow :** Tested UI flow with error scenarios, gracefully handled errors on UI
-   [x] **Common component changed :** Tested change in common component across views to prevent breaking elsewhere.

### CODE REVIEW

File: src/assets/homepageassets/net-design.svg
Comment (ram-primathon) [inline src/assets/homepageassets/net-design.svg:1]: remove this SVG and use CSS to create this

File: src/assets/homepageassets/our-work-1.png
Comment (ram-primathon) [inline src/assets/homepageassets/our-work-1.png:1]: it's a big size image also why do we need a full image?

File: src/assets/homepageassets/our-work-2.png
Comment (ram-primathon) [inline src/assets/homepageassets/our-work-2.png:1]: big size image

File: src/assets/homepageassets/our-work-3.png
Comment (ram-primathon) [inline src/assets/homepageassets/our-work-3.png:1]: Very big size image

File: src/assets/homepageassets/our-work-4.png
Comment (ram-primathon) [inline src/assets/homepageassets/our-work-4.png:1]: same

File: src/index.css [hunk 1]
```diff
@@ -1,13 +1,11 @@
-@import url('https://fonts.googleapis.com/css2?family=DM+Serif+Display:ital@0;1&family=DM+Serif+Text:ital@1&family=Poppins:ital@0;1&family=Roboto+Condensed:ital,wght@0,100..900;1,100..900&family=Roboto:ital@0;1&display=swap');
-@import url('https://fonts.googleapis.com/css2?family=DM+Serif+Display:ital@0;1&family=DM+Serif+Text:ital@1&family=Poppins:ital@0;1&family=Roboto+Condensed:ital,wght@0,100..900;1,100..900&family=Roboto:ital@0;1&display=swap');
-@import url('https://fonts.googleapis.com/css2?family=DM+Serif+Display:ital@0;1&family=DM+Serif+Text:ital@1&family=Poppins:ital@0;1&family=Roboto+Condensed:ital,wght@0,100..900;1,100..900&family=Roboto:ital@0;1&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=DM+Serif+Display:wght@400;500;600;700&family=DM+Serif+Text:ital,wght@1,400;1,500;1,600&family=Poppins:wght@400;500;600;700&family=Roboto+Condensed:wght@400;500;600;700&family=Roboto:ital,wght@0,400;0,500;0,600;0,700&display=swap');
```
Comment (ram-primathon) [inline src/index.css:1]: What was the issue here? why did you hve remove this?

File: src/pages/homepage/WorkComponent.jsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
+import { Fragment, useRef } from 'react';
+import { useGSAP } from '@gsap/react';
+import { NavLink } from 'react-router-dom';
+import backgroundDesign from '../../assets/homepageassets/net-design.svg';
```
Comment (ram-primathon) [inline src/pages/homepage/WorkComponent.jsx]: Please don't import images like this it should come from one js file

File: src/pages/homepage/WorkComponent.jsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+                    <NavLink
+                        to="/"
+                        className="flex items-center justify-between bg-gray-bg-color py-2 pr-2 pl-10 w-64 md:w-72 lg:w-80 2xl:w-95 rounded-full hover:bg-gradient-to-r hover:from-gray-bg-color hover:from-30% hover:to-pink-bg-color">
+                        <h1 className="text-portfolio-button-color text-xs md:text-sm xl:text-base font-semibold">
+                            EXPORT PORTFOLIO
+                        </h1>
+                        <div className="flex w-10 h-10 xl:w-14 xl:h-14">
+                            <img
+                                src={portfolioArrow}
+                                alt="arrow icon"
```
Comment (ram-primathon) [inline src/pages/homepage/WorkComponent.jsx]: what is arrow icon?

**Review by ram-primathon — CHANGES_REQUESTED** [review]

File: src/index.css (hunk 1 above)
Comment (TheSahilVerma) [reply src/index.css:1]: Font is not working properly Its medium light lighter is not working but after the code change this works properly now

File: src/pages/homepage/WorkComponent.jsx (hunk 2 above)
Comment (TheSahilVerma) [reply src/pages/homepage/WorkComponent.jsx]: Done sir make js file in assets

File: src/assets/homepageassets/net-design.svg
Comment (TheSahilVerma) [reply src/assets/homepageassets/net-design.svg:1]: removed but not added css not working appropriately

File: src/pages/homepage/WorkComponent.jsx (hunk 2 above)
Comment (TheSahilVerma) [reply src/pages/homepage/WorkComponent.jsx]: image.js

File: src/pages/homepage/WorkComponent.jsx (hunk 3 above)
Comment (TheSahilVerma) [reply src/pages/homepage/WorkComponent.jsx]: Changed name

**Pull request closed without merging**